src/test/java/org/folio/rest/api/PatronActionSessionAPITest.java

```diff
@@ -179,12 +185,99 @@ public void cannotUpdateNonExistentActionSession() throws InterruptedException,
     assertThat(response.getStatusCode(), is(404));
   }
 
-  private JsonObject createSession(String actionType){
+  @Test
+  public void canGetPatronActionSessionStorageExpiredSessionPatronIds() throws InterruptedException,
+    ExecutionException, TimeoutException, MalformedURLException {
+
+    String firstPatronId = UUID.randomUUID().toString();
+    createEntity(firstPatronId, "Check-in", DateTime.now().minusDays(3));
+    createEntity(firstPatronId, "Check-out", DateTime.now().minusDays(3));
+    String secondPatronId = UUID.randomUUID().toString();
+    createEntity(secondPatronId, "Check-in", DateTime.now().minus(1));
+    createEntity(secondPatronId, "Check-out", DateTime.now().minus(1));
+    createEntity(secondPatronId, "Check-out", DateTime.now().minus(1));
+
+    assertThat(getExpiredSessionPatronIds("Check-in", 10, DateTime.now().minusDays(2)).size(),
+      is(1));
+    assertThat(getExpiredSessionPatronIds("Check-out", 10, DateTime.now()).size(),
+      is(2));
+    assertThat(getExpiredSessionPatronIds("Check-in", 10, DateTime.now().minusDays(2)).get(0),
+      is(firstPatronId));
+    assertThat(getExpiredSessionPatronIds("Check-out", 10, DateTime.now().minusDays(2)).get(0),
+      is(firstPatronId));
+    assertThat(getExpiredSessionPatronIds("Check-out", 2, DateTime.now()).size(),
```

Comment:
There is no difference between this and 2nd case
If it's intended to check "limit" parameter, it's better to pass limit=1

Reply:
Thanks, the test has been updated